Lib/fontParts/base/bPoint.py

```diff
@@ -24,6 +24,14 @@ def __eq__(self, other):
             return self._point == other._point
         return False
 
+    def __ne__(self, other):
+        if hasattr(other, "_point"):
+            return self._point != other._point
+        return False
+
+    def __hash__(self):
+        return id(self)
```

Comment:
I think this is still a problem: two different objects may still compare equal through their `._point` attr, yet will have a different hash.